packages/playground/website/vite.config.ts

```diff
@@ -151,6 +152,10 @@ export default defineConfig(({ command, mode }) => {
 					}
 				},
 			} as Plugin,
+			/**
+			 * Merge cache manifest files into a single `cache-files.json` file.
```

Comment:
Should it say this instead?

```suggestion
			 * Merge cache manifest files into a single `website-cache-files.json` file.
```

Reply:
No, [the file it generates](https://github.com/WordPress/wordpress-playground/blob/966f7bf3a10f2b074fe2c2abe4a7298c909c432a/packages/vite-extensions/vite-website-cache-paths.ts#L51) is `cache-files.json`.

[This manifest line](https://github.com/WordPress/wordpress-playground/pull/1573/files#diff-605dc867822b052f4f258251c03a82b831c755e64a70872763b32635ec5b6b87R186) isn't needed and I removed it [9f1ad7c](https://github.com/WordPress/wordpress-playground/pull/1573/commits/9f1ad7c68c3867a41c094bd8f29c7255678ca223)